### ISSUE TITLE
[Bug] Parsing errors with Mardown lists

### ISSUE THREAD

**Issue by IgnacioHeredia** [graphlab-fr/cosma#86]
### What version of Cosma are you using?

CLI 2.0.0

### On which operating system(s) did you observe the bug?

- [ ] macOS
- [ ] Windows
- [X] Linux

### What version of the operating system are you on?

Ubuntu 20.04

### Actions leading to the bug

Described below.

### Bug description

I think there is something wrong with how markdown lists are parsed. Let me explain with three different examples:

1) Big list ✅ 
```
* element 1
* element 2
* element 3
* element 4
```

![image](https://github.com/graphlab-fr/cosma/assets/16413205/6fdf072d-6f40-4425-8094-5fcfe7503a84)

2) Two small lists, separated by a new line 🔴 --> both lists appear broken!

```
* element 1
* element 2

* element 3
* element 4
```

![image](https://github.com/graphlab-fr/cosma/assets/16413205/510fa5a1-0ead-4937-bddd-e4f9f6248c49)

3) Two small lists, separated by text  ✅  --> both list are correctly formatted
```
* element 1
* element 2
Demo:
* element 3
* element 4
```

![image](https://github.com/graphlab-fr/cosma/assets/16413205/8a7f07b6-a2e0-4bbf-8817-af7c0ee91f5c)

Now, regarding example 3), while lists are correctly formatted, I not sure the formatting is entirely correct either, because the intermediate text shouldn't be a completely newline, it should be indented as to belong to the previous bullet point. It seems that Cosma doesn't properly parse [multilines in bullet lists](https://stackoverflow.com/questions/33673639/multi-line-bullet-list-in-markdown).

Let me show with an example:
```
* element 1
more info about element 1
* element 2 \
more info about element 2
* element 3
```

**Github**: Text is properly indented.
* element 1
more info about element 1
* element 2 \
more info about element 2
* element 3

**Cosma**:
None of the options work, text is never indented so as to belong to the previous bulletpoint. It's just treated as text separating different lists, not as a single list.
![image](https://github.com/graphlab-fr/cosma/assets/16413205/6adab692-2d09-4b9b-bd5a-9b240a686899)

**Comment by infologie** [comment]
Thanks @IgnacioHeredia. This is not a bug but the unintended consequence of a CSS declaration:

https://github.com/graphlab-fr/cosma/blob/260ee527d52368232bb74a009f196d3efbda44ab/core/static/styles/styles.css#L532-L536

This makes it so that lists are flush with the margin, with the bullet "inside" the paragraph instead of "outside" on the left.

I don't know why this is the way it is… I rarely use lists, so I never noticed this until you pointed it out.

I'm happy to modify our CSS so that lists are typeset more conventionally. It's an easy fix, so it will be in the next update.

**Edit:** well, actually it is a bug, since it prevents multiline bulleted lists from being rendered properly. Anyway, an easy bug to squash!

**Comment by infologie** [comment]
@IgnacioHeredia Fixed in [v2.3.0](https://github.com/graphlab-fr/cosma/releases/tag/2.3.0).
